From 03b9e96bd39b47752a24be012de9f99854e20eb8 Mon Sep 17 00:00:00 2001
From: Jiaxun Yang <jiaxun.yang@flygoat.com>
Date: Fri, 28 Oct 2022 21:31:01 +0100
Subject: [PATCH 5/6] HACK: target/mips: Enable MTTCG

Didn't find root cause of underlying issue but it looks fine to
enable it for now.

Signed-off-by: Jiaxun Yang <jiaxun.yang@flygoat.com>
---
 configs/targets/mips64el-softmmu.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/targets/mips64el-softmmu.mak b/configs/targets/mips64el-softmmu.mak
index 5a52aa4b64..35389733ab 100644
--- a/configs/targets/mips64el-softmmu.mak
+++ b/configs/targets/mips64el-softmmu.mak
@@ -1,3 +1,4 @@
 TARGET_ARCH=mips64
 TARGET_BASE_ARCH=mips
 TARGET_NEED_FDT=y
+TARGET_SUPPORTS_MTTCG=y
-- 
2.39.1

